contracts/package.json

```diff
@@ -16,9 +16,9 @@
     "compile": "hardhat compile",
     "coverage": "hardhat coverage",
     "prepublishOnly": "pnpm compile && ./scripts/prepublish_generate_abi_folder",
-    "publish-beta": "pnpm publish --tag beta",
+    "publish- beta": "pnpm publish --tag beta",
```

Comment:
I would have to assume this is unintentional? it will likely break github release workflow

Reply:
yea that's definitely unintentional. Idk how that got changed but i'll revert it